gateway-ha/src/test/java/io/trino/gateway/ha/router/TestTrinoRequestUser.java

```diff
@@ -42,4 +52,34 @@ void testJsonCreator()
         assertThat(deserializedTrinoRequestUser.getUser()).isEqualTo(trinoRequestUser.getUser());
         assertThat(deserializedTrinoRequestUser.getUserInfo()).isEqualTo(trinoRequestUser.getUserInfo());
     }
+
+    @Test
+    void testUserFromJwtToken()
+    {
+        final String USER_NAME_CLAIM = "username";
+        final String USER_NAME_VALUE = "trino";
```

Comment:
We usually don't use `final` and uppercase for local variables in Trino project.
```suggestion
        String claimName = "username";
        String claimValue = "trino";
```